openvdb/openvdb/tree/Tree.h

```diff
@@ -361,7 +361,7 @@ class Tree: public TreeBase
     Index64 activeTileCount() const override { return tools::countActiveTiles(*this); }
 
     /// Return the minimum and maximum active values in this tree.
-    void evalMinMax(ValueType &min, ValueType &max) const;
+    void evalMinMax(ValueType &min, ValueType &max) const { tools::minMaxValues(*this, min, max); }
```

Comment:
Thought about this a bit more, I don't think it's worth sticking with the single threaded option on `evalMinMax` if we're not also doing that for other methods such as `activeVoxelCount` as that just isn't consistent. So as long as we mark this method as deprecated then perhaps this change is fine. We can still discuss in TSC but let's get this in as is (+ the deprecation).

@ghurstunither I notice there's CI failures, are you able to take a look and fix those?

Reply:
Thanks for pointing out the unit tests! I was not correctly supporting edge cases like empty grids. This should be fixed now.

Reply:
Regardless of what's best here in this PR, _should_ `evalMinMax` and `activeVoxelCount`, etc introduce an optional threading argument? I'm not suggesting this would solve the problems referred to above, but is there harm adding this switch? It would be the easiest way for one to disable multithreading if desired.

Reply:
Given that we're looking to deprecate `Tree::evalMinMax()`, I would think that it's not necessary, but (and I think we'll talk about this in the next TSC), perhaps adding a comment to note that this method is now multi-threaded and to switch to use your new tools one if the single-threaded behavior is desirable?